### ISSUE TITLE
Bug Report: Note Kinds Custom field placeholder actually fills in field

### ISSUE THREAD

**Issue by Hundrec** [FunkinCrew/Funkin#3608]
### Issue Checklist

- [X] I have properly named my issue
- [X] I have checked the Issues/Discussions pages to see if my issue has already been reported

### Platform

Itch.io (Downloadable Build) - Windows

### Browser

None

### Version

0.5.1

### Description (include any images, videos, errors, or crash logs)

When clicking into the Custom Note Type field, "Kind..." lights up and turns into actual text that needs to be deleted.

![image](https://github.com/user-attachments/assets/62dbe99b-1b41-4501-9be7-87886f7318d5)

### Steps to Reproduce

1. Open the Chart Editor
2. Click on "Window"
3. Open the Notes tab
4. Select "Custom" from the dropdown menu
5. Click into the Custom Note Type field

**Comment by Lasercar** [comment]
Yeah, this is a issue with haxe UI. There's this placeholder value that's only displayed when it's empty and is supposed to clear away when you click into it, but setting the value back to null brings it back, but doesn't clear the placeholder again when you click into it.

**Comment by Starexify** [comment]
Dont think this is an issue anymore since haxeui updated, or we used the actual `placeholder` field later versions.

**Comment by Hundrec** [comment]
<img width="311" height="264" alt="Image" src="https://github.com/user-attachments/assets/5e63609a-5594-42b5-a168-aa114bc82625" />

Just got it to happen on the latest 0.8.3 `develop` by switching around between notekinds in the dropdown.

Still don't know what exactly causes this to happen.

**Comment by Starexify** [comment]
I managed to find the reason: refreshing the toolbox keeps `CUSTOM` or the old kind from the previously selected notekind, if you select a default note, it would use custom and fill it with the placeholder text. I tried to fix it in the `NoteKindToolbox` or smth like that and I couldn't fix it.
